### DISCUSSION TITLE
Quartz .NET wtih ADOJobStore how to check if job is created?

### DISCUSSION THREAD

**Discussion by lvyyljn** [quartznet/quartznet#2645] [Q&A]
I have a Quartz.NET application with ADOJobStore We create our jobs(Cron jobs that execute once a day) on a application start within AddQuartz using AddJob/AddTrigger It seems like on each restart jobs are updated in DB (as expected) and it looks like because of this our misfire policy doesn't work.

We have configured misfire with WithMisfireHandlingInstructionFireAndProceed, I started app, then closed it, started it again only after NEXT_FIRE_TIME in DB + 1 minute, but job wasn't triggered.

Basically my question is - if there's a way to know that job is already created during AddQuartz?
